SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj

```diff
@@ -404,71 +419,76 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		80D0490B2668312D00B53FDE /* XCRemoteSwiftPackageReference "braintree_ios" */ = {
+		9C1FC8ED27CFE11500E6E14D /* XCRemoteSwiftPackageReference "braintree_ios" */ = {
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/braintree/braintree_ios";
 			requirement = {
-				branch = master;
+				branch = "create-sepa-module";
```

Comment:
We have a JIRA ticket to put this back to master when the feature branch is merged